subsys/bluetooth/mesh/priv_beacon.h

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+int bt_mesh_priv_beacon_srv_state_store(void);
```

Comment:
Why not use `pending_store` callback and `bt_mesh_model_data_store_schedule`: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/bluetooth/mesh/access.h#L852-L860 ?

Reply:
I will check it out 👍

Reply:
Yes I think this is a better way, thanks